doc/jobs/prerelease_jobs.rst

```diff
@@ -101,3 +112,32 @@ scripts which can be invoked manually in the same order:
   workspace.
   It also invokes the tool ``catkin_test_results --all`` to output a
   summary of all tests in the overlay workspace.
+
+Run the *prerelease* job on Travis
+----------------------------------
+
+Since it is easy to run a *prerelease* job locally it can also be run on Travis to either test every commit or pull request.
+The setup and invocation is the same as locally.
+The `.travis.yml <https://github.com/ros-infrastructure/ros_buildfarm/blob/master/.travis.yml>`_ file of the *ros_buildfarm* repository is a good starting point.
+
+Run for "custom" repositories
+-----------------------------
+
+Most job types require that the tested repository is being listed in a ROS distribution file.
+For the *prerelease* job that is not necessary.
+Any repository containing ROS packages can be processed by this job.
+
+Since the ROS distribution doesn't know about the repositories several pieces
+of information have to be passed using the command line argument
+`--custom-repo`.
+The following parts are all separated by colons: the repository name, the
+repository type, the repository URL, the branch or tag name, e.g.:
+
+.. code:: sh
+
+  generate_prerelease_script.py ... my_repo_name ... --custom-repo \
```

Comment:
Is `my_repo_name`  required here? With regard to your other [comment](https://github.com/ros-infrastructure/ros_buildfarm/pull/401/files/7e1df874a031c4a5eb22a154c3dffceec047c39e?diff=unified#r106282905)